dynamic_env_pytest_tests/tests/services/http_gate/test_http_bearer.py

```diff
@@ -83,15 +83,23 @@ def bearer_token_no_limit_for_others(self, user_container: str) -> str:
             )
             return bearer_token_base64_from_file(bearer_signed)
 
+    @pytest.fixture(scope="class", params=["HTTP", "REST"])
+    def gw_endpoint(self, request):
```

Comment:
Does it work this way? If it does - that's good. If not, another way is this:
Define a special pytest hook method (put it into conftest.py)
```
def pytest_generate_tests(metafunc):
    if "gw_endpoint" in metafunc.fixturenames:
        metafunc.parametrize("gw_endpoint", ["HTTP, "REST"], indirect=True)
```
And then this fixture will be the same, but without params and you can also define it in conftest.py